src/test/java/bio/overture/ego/service/UserServiceTest.java

```diff
@@ -55,7 +57,7 @@
 @RunWith(SpringRunner.class)
 @ActiveProfiles("test")
 @Transactional
-@Ignore("replace with controller tests.")
+// @Ignore("replace with controller tests.")
```

Comment:
Way back when, there were issues with Service level tests, due to weird errors related to business logic happening in a persistence context. To fix these issues, we  ignored this test (mostly because we wanted it to be evident) and replaced the tests with controller tests. 

The `testGetManyUsersWithRelations` test is a legit test of serivce functionality and should be tested. Please delete this comment, and remove all tests except your `testGetManyUsersWithRelations` test.

Reply:
Deleted like everything. Felt cathartic.